feature-libs/checkout/_index.scss

```diff
@@ -1 +1,9 @@
-@import './styles/index';
+@import '~@spartacus/styles/scss/core';
+
+@import '~bootstrap/scss/functions';
+@import '~bootstrap/scss/variables';
+@import '~bootstrap/scss/mixins';
+
+@import './base/index';
+@import './b2b/index';
+@import './scheduled-replenishment/index';
```

Comment:
This file is common for both the new and old checkout?
Does this mean the old checkout will be using the new checkout's styles?
Does it also mean the old checkout will have to pull the whole new checkout just for the styles?

Reply:
so how styles work with our Spartacus architecture setup is that all styles gets bundled in the end in the main entry-point of styles. We just separate them in libraries for ease of use and understanding I believe. So old checkout will still look the same as what it was before.

Basically, instead of having the stylesheet in one styles folder, it is just separated per library for the corresponding components. In the end, it still ends up being the same as what it was before, but more decoupled. Also, when we use schematics, we pull from the main styles of the 'feature' lib, which contains everything from before.